src/components/Modal/components/CloseButton/CloseButton.tsx

```diff
@@ -11,15 +12,23 @@ export interface Props {
   onClick(): void;
 }
 
-export default function CloseButton({title = true, onClick}: Props) {
+type CombinedProps = Props & WithAppProviderProps;
+
+function CloseButton({title = true, onClick, polaris: {intl}}: CombinedProps) {
   const className = classNames(
     styles.CloseButton,
     !title && styles.withoutTitle,
   );
 
   return (
-    <button onClick={onClick} className={className}>
```

Comment:
Looks like this is being addressed in PR #1582 (which I happened to get to first today).

Reply:
Would it make sense to close https://github.com/Shopify/polaris-react/pull/1582 if the same thing is already done here? :)

Reply:
@robin-drexler i'll merge your PR into here :)